.github/actions/verify-dotnet-consolidation/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Verify that .NET packages are consolidated
+description: >
+  Verifies that the NuGet packages of a .NET solution are consolidated, i.e. the same version of a given package is used
+  in all projects.
+
+inputs:
+  directory:
+    required: false
+    default: .
+    description: Path to the directory where a solution file can be found.
+  solution-path:
+    required: false
+    default: "*.sln"
+    description: >
+      The path of the solution file to be processed. The value is given to PowerShell's `Get-ChildItem` cmdlet as-is, so
+      grepping is still possible but a name with spaces must be escaped separately.
```

Comment:
```suggestion
      The path of the solution file is to be processed. The value is given to PowerShell's `Get-ChildItem` cmdlet as-is, so
      grepping is still possible but a name with spaces must be escaped separately.
```
`Grammarly` not me

Reply:
Grammarly is wrong in this case :). If the sentence was alone, it would make sense, but not here. If this would be out of the context of the action parameter, the sentence would be: "`solution-path` is a parameter for the path of the solution file to be processed." This is a truncated sentence in the comment.